content/en/docs/concepts/configuration/secret.md

```diff
@@ -201,7 +201,8 @@ to create a Docker registry Secret, you can do:
 kubectl create secret docker-registry secret-tiger-docker \
   --docker-username=tiger \
   --docker-password=pass113 \
-  --docker-email=tiger@acme.com
+  --docker-email=tiger@acme.com \
+  --docker-server=my-registry:5000
```

Comment:
A Docker registry's server is a hostname, right?
```suggestion
  --docker-server=my-registry.example:5000
```

Reply:
Added a commit for the same. For the JSON dump output i will add it accordingly asap
Thanks